ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/91521_SCAPP_apply-to-become/91489_Apply-to-become-get-application-data-map.js

```diff
@@ -0,0 +1,402 @@
+/// <reference types ='Cypress'/>
+import {data} from "../../fixtures/cath121-body.json"
+
+describe('91489: Apply-to-becom GET data application types', () => {
+    let url = Cypress.env('url')
+    let dataAppSch = data.applyingSchools[0]
+    
+    afterEach(() => {
+        cy.storeSessionData()
+    });
+    
+    before(() => {
+        cy.login()
+        cy.get('[id="school-name-44"]').click()
+        cy.visit(url+'/school-application-form/519')
+    });
+
+    after(() => {
+        cy.clearLocalStorage()
+    });
+
+    // Overview
+    it('TC01: Overview - Application Lead', () => {
+        cy.get('[test-id="Overview1_key"]').should('contain.text', 'Application to join')
+        cy.get('[test-id="Overview1_value"]').should('contain.text', data.name + ' with ' + dataAppSch.schoolName)
+        cy.get('[test-id="Overview2_key"]').should('contain.text', 'Application reference')
+        cy.get('[test-id="Overview2_value"]').should('contain.text', data.applicationId)
+        cy.get('[test-id="Overview3_key"]').should('contain.text', 'Lead applicant') 
+        cy.get('[test-id="Overview3_value"]').should('contain.text', data.applicationLeadAuthorName)
+    })
+
+    // Details of Trust
+    it('TC02: Trust details', () => {
+        cy.get('[test-id="Overview_Details1_key"]').should('contain.text', 'Trust name')
+        cy.get('[test-id="Overview_Details1_value"]').should('contain.text', data.name)
+        // 'Upload evidence' IS NOT BE COVERED IN THIS TEST AS THESE ARE UPLOAD LINKS - [test-id="Overview_Details2_key"]
+        cy.get('[test-id="Overview_Details3_key"]').should('contain.text', 'Will there be any changes to the governance of the trust due to the school joining?') 
+        cy.get('[test-id="Overview_Details3_value"]').should('contain.text', 'No')
+        cy.fixture('cath121-body.json').as('userData').then((userData) => {
+            expect(userData.data.changesToLaGovernance).to.eq(false)
+        })
+        cy.get('[test-id="Overview_Details4_key"]').should('contain.text', 'Will there be any changes at a local level due to this school joining?') 
+        cy.get('[test-id="Overview_Details4_value"]').should('contain.text', 'No')
+        cy.fixture('cath121-body.json').as('userData').then((userData) => {
+            expect(userData.data.changesToTrust).to.eq(false)
+        })
```

Comment:
We agreed that we wanted to check that what you see on the front end matches that on the returned data. I am using dot notation to invoke the data from the JSON file. Numbers, Dates, Booleans are invoked with additional methods because the data appears as a 'strings' on the Frontend. 

Writing them dynamically would not actually solve the issue in which we are trying to solve at this very point. This test is only to cover your specific concerns in terms of 'Does the data match the cells'.

I can look at writing these dynamically, but a task will have to be written up and further more, to write some tests dynamically we would have to spend time to review which questions are a follow up answers - this potentially would need a whole different approach.